features/FEATURE_LWIP/lwip-interface/mbed_lib.json

```diff
@@ -16,7 +16,11 @@
         "addr-timeout": {
             "help": "On dual-stack system how long to wait preferred stack's address in seconds",
             "value": 5
-        },
+        },        
+        "both-addr-timeout": {
```

Comment:
Maybe comment to say that it's not terribly meaningful to set both options. How about a boolean that changes what addr-timeout means - extra wait for preferred, or extra wait for all addresses. Make the help clear that is the extra wait on top of the initial wait for any address.